src/core.js

```diff
@@ -238,8 +238,8 @@ jQuery.extend( {
 	},
 
 	// Evaluates a script in a global context
-	globalEval: function( code ) {
-		DOMEval( code );
+	globalEval: function( code, nonce ) {
```

Comment:
I don't like the signature, one could easily imagine a realm parameter occupying that position. And I don't really feel great about an options object here, but maybe that is the best approach. :\

Reply:
Oh, the last version doesn't work as we still need the `getAttribute` fallback for non-nonce params. But https://github.com/mgol/jquery/commit/e49e3e865bbab856fe67b6dd75aa844643badbc5 is fine.

Reply:
I prefer the options object, even if there's only one option.